### PR TITLE
Update Integration tests to new format and for new handlers

### PR DESCRIPTION
**Pull request by evangrayk**
Add integration tests for NotFound handler, status handler, default handlers, and use new config syntax.

### CODE REVIEW

**Comment by evangrayk** [comment]
We also now use integration tests in Travis CI so that we are sure our server is fully test-compliant in master.

File: tests/integration.py [hunk 1]
```diff
@@ -182,15 +192,118 @@ def test_basic():
 
                 return Test.PASS
 
+    def test_not_found():
```
Comment (Filisha) [inline tests/integration.py:195]: 👍

File: tests/integration.py [hunk 2]
```diff
@@ -182,15 +192,118 @@ def test_basic():
...
+                        'NotFoundHandler',
+                        }
+
+                for chunk in expectedOutputChunks:
+                    if chunk not in content:
+                        return "response does not contain expected '%s'" % chunk
+
+                return Test.PASS
+
+    def test_default():
```
Comment (Filisha) [inline tests/integration.py:270]: 👍

**Review by Filisha — APPROVED** [review]
Amazing work. 
I was wondering if we needed to change the format of:
"root tesFiles1" to something more suited instead of saying testFiles1

**Comment by evangrayk** [comment]
@Filisha yeah I just named the folder of testing files 'testFiles1' for lack of a better name... We should think of a better name haha. I'll update that later.

**Pull request merged**